instrumentation/rocketmq-client-4.8/testing/src/main/java/base/BaseConf.java

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package base;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.log4j.Logger;
+import org.apache.rocketmq.broker.BrokerController;
+import org.apache.rocketmq.client.producer.TransactionListener;
+import org.apache.rocketmq.common.MQVersion;
+import org.apache.rocketmq.namesrv.NamesrvController;
+import org.apache.rocketmq.remoting.protocol.RemotingCommand;
+import org.apache.rocketmq.test.client.rmq.RMQAsyncSendProducer;
+import org.apache.rocketmq.test.client.rmq.RMQNormalConsumer;
+import org.apache.rocketmq.test.client.rmq.RMQNormalProducer;
+import org.apache.rocketmq.test.client.rmq.RMQTransactionalProducer;
+import org.apache.rocketmq.test.clientinterface.AbstractMQConsumer;
+import org.apache.rocketmq.test.clientinterface.AbstractMQProducer;
+import org.apache.rocketmq.test.factory.ConsumerFactory;
+import org.apache.rocketmq.test.listener.AbstractListener;
+import org.apache.rocketmq.test.util.MQAdmin;
+import org.apache.rocketmq.test.util.MQRandomUtils;
+
+public class BaseConf {
+  public static final String nsAddr;
+  public static final String broker1Addr;
+  protected static String broker1Name;
+  protected static final String clusterName;
+  protected static int brokerNum;
+  protected static final NamesrvController namesrvController;
+  protected static final BrokerController brokerController1;
+  protected static final List<Object> mqClients = new ArrayList<Object>();
+  protected static boolean debug = false;
+  private static final Logger log = Logger.getLogger(BaseConf.class);
+
+  static {
+    System.setProperty(
+        RemotingCommand.REMOTING_VERSION_KEY, Integer.toString(MQVersion.CURRENT_VERSION));
+    namesrvController = IntegrationTestBase.createAndStartNamesrv();
+    nsAddr = "127.0.0.1:" + namesrvController.getNettyServerConfig().getListenPort();
+    brokerController1 = IntegrationTestBase.createAndStartBroker(nsAddr);
+    clusterName = brokerController1.getBrokerConfig().getBrokerClusterName();
+    broker1Name = brokerController1.getBrokerConfig().getBrokerName();
+    broker1Addr = "127.0.0.1:" + brokerController1.getNettyServerConfig().getListenPort();
+    brokerNum = 2;
+  }
+
+  public BaseConf() {}
```

Comment:
All methods & fields in this class are static, can you make the constructor private (and class final) and call static methods everywhere without creating a `BaseConf` instance?